broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/LogStreamPartitionStep.java

```diff
@@ -95,8 +94,9 @@ public String getName() {
     if (raftTerm != context.getCurrentTerm()) {
       return left(buildWrongTermException(context, raftTerm));
     } else {
-      logStorage = AtomixLogStorage.ofPartition(server::openReader, logAppender);
+      final var logStorage = AtomixLogStorage.ofPartition(server::openReader, logAppender);
       server.addCommitListener(logStorage);
+      context.setLogStorage(logStorage);
```

Comment:
:wrench: I would prefer if we set the log storage in the method that calls this place. We could also remove the `context` parameter altogether if we pass in `currentTerm` and `partitionId`

The reason I don't want it here, is that hierarchically, I would read the open method first. And there I want to see all the changes to the context.

Reply:
sounds good